app/main/views/audits.py

```diff
@@ -83,23 +90,43 @@ def list_audits():
     if object_type:
         if object_type not in AUDIT_OBJECT_TYPES:
             abort(400, 'invalid object-type supplied')
-        if not object_id:
-            abort(400, 'object-type cannot be provided without object-id')
```

Comment:
The check isn't being removed - the *ability* is being *added*. There *is* a reason for it. And I should have written down that reason when I did it, because I can't *quite* remember it now...